.ci/pipeline/ci.yml

```diff
@@ -43,6 +43,8 @@ jobs:
       . /usr/share/miniconda/etc/profile.d/conda.sh
       conda activate CB
       ./conda-recipe/run_test.sh
+      pip uninstall -y scipy scikit-learn
+      python -c "import daal4py.sklearn"
```

Comment:
shouldn't this be in script instead of direct specification in pipeline ?

Reply:
Now I remove this lines, because daal4py.sklearn assumes the presence of scikit-learn module.